docs/topics/using-openrewrite-recipes.adoc

```diff
@@ -49,11 +49,11 @@ You can include more than one recipe by specifying each in the `activeRecipes` p
 ** `dryRun` : The script returns a list of proposed changes. Ignore the `"Run 'mvn rewrite:run' to apply the recipes"` message.
 ** `run`: The script applies the changes.
 
-. Run `windup-cli` with `--goal run` to apply the recipe:
+. Run `{mta-cli}` with `--goal run` to apply the recipe:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ ./windup-cli --openrewrite --input </path/to/source/project> \
+$ ./{mta-cli} --openrewrite --input </path/to/source/project> \
```

Comment:
The `--openrewrite` argument doesn't exist anymore. For code transformation the command is now:

```bash
mta-cli transform openrewrite ...
```